saisdkdump/saisdkdump.cpp

```diff
@@ -12,6 +12,8 @@ extern "C" {
 #include <sai.h>
 }
 
+sai_status_t sai_dbg_generate_dump(const char *c) {return 0; }
```

Comment:
please remove this, since this should be already implemented in new SAI, i added this manually on my branch since my SAI didn't implemented that yet, actually it not need to be implemented, just exported for linking